.gitignore

```diff
@@ -22,3 +22,6 @@ docs/site/
 # committed for packages, but should be committed for applications that require a static
 # environment.
 Manifest.toml
+
+# Housekeeping files generated by Visual Studio Code
+.vscode/settings.json
```

Comment:
Maybe better just to make this `.vscode`, since there is no reason anything in that directory should make it into version control.

Reply:
Good idea. Done!